src/digital.rs

```diff
@@ -1,6 +1,7 @@
 //! Digital I/O
 
 /// Single digital push-pull output pin
+#[cfg(not(feature = "use-fallible-digital-traits"))]
```

Comment:
Is it possible to mark all the infallible traits as depreciated with a suggestion to use the fallible ones?

Reply:
done